src/styles/global.scss

```diff
@@ -13,8 +13,7 @@
 @import './variables';
 
 ::-webkit-scrollbar {
-  width: 1px;
-  background-color: transparent;
+  display: none;
```

Comment:
Probably, it breaks accessibility in some cases (when not ios?), but scrollbars are already invisible.

Reply:
We already have scrollbar hidden in firefox https://github.com/aeternity/superhero-wallet/blob/6f49b24c2599911f493b52c3d1b853b37ca9068f/src/styles/global.scss#L197
let's keep it consistent